http/server.lua

```diff
@@ -15,6 +16,21 @@ local errno = require 'errno'
 
 local DETACHED = 101
 
+-- A table that maps fiber object to time when it was active last time.
+-- Using for tracking idle connections.
+local connections = {}
+
+local function close_idle_connections(connections, idle_timeout)
+    -- FIXME: complexity is O(N) and it is bad
+    for fiber_obj, start_time in pairs(connections) do
+        if fiber.clock() - start_time >= idle_timeout then
```

Comment:
Probably we could calculate this value only once at start of the loop.
And also you can use queue instead of array here to solve fixme.

Reply:
Rewrote patch with using `timeout` in `tcp_server`.